src/store/middleware/inputMiddleware.js

```diff
@@ -25,6 +27,24 @@ const inputMiddleware = store => next => action => {
 
     return result;
   } else {
+    switch (action.type) {
+      case enableIK.type: {
+        store.dispatch(messageRover({
+          message: {
+            type: "enableInverseKinematics",
+            enabled: useSelector(selectInverseKinematicsEnabled)
+          }
+        }));
+        break;
+      }
+
+      case (roverDisconnected.type || roverConnected.type): {
```

Comment:
Is this the right way to check for multiple cases? This looks like it'll use type coercion to get truthy or falsey values and check the action type against that.

To check for multiple cases you should do:
```
case roverDisconnected.type: case roverConnected.type: {
    ...
}
```